### PR TITLE
Add bool dtype support for PLY files

### PR DESCRIPTION
**Pull request by Nicholas-Mitchell**
This PR addresses https://github.com/daavoo/pyntcloud/issues/309 and adds support for the file reader to deal with Boolean fields (writing boolean fields was already not an issue). It also adds a new test fixture file and a simple unit test.

The new test data PLY file contains the following header:
diamond with bool points dtypes

which loads to give the following dataframe - notice the newly inserted `is_green` column of type `bool`:

green blue nx ny nz is green

### CODE REVIEW

**Review by daavoo — APPROVED** [review]
Thanks @Nicholas-Mitchell 🚀 !

File: pyntcloud/io/ply.py [hunk 1]
```diff
@@ -242,7 +248,7 @@ def describe_element(name, df):
     -------
     element: list[str]
     """
-    property_formats = {'f': 'float', 'u': 'uchar', 'i': 'int'}
+    property_formats = {'f': 'float', 'u': 'uchar', 'i': 'int', 'b': 'bool'}
```
Comment (daavoo) [inline pyntcloud/io/ply.py:251]: Could we add a test for `write_ply` with bool?

**Review by daavoo — CHANGES_REQUESTED** [review]
Just minor thing. Adding test for `write_ply`

**Comment by Nicholas-Mitchell** [comment]
@daavoo - I've added a test that writes a PLY file containing Boolean data

https://github.com/daavoo/pyntcloud/pull/321/files?authenticity_token=oqEfG4St1pVEVUuZtjJvNZLORmUQnhhvJf%2BqJZLj2hUWQxRKGEq7MUcgaKZUuxi40njLo2jnNC8%2BBlap5EDukA%3D%3D&file-filters%5B%5D=.py#diff-021377fee185787af295603466e9f1f3fec039246017a515213eb7c2f2115888

File: pyntcloud/io/ply.py (hunk 1 above)
Comment (Nicholas-Mitchell) [reply pyntcloud/io/ply.py:251]: https://github.com/daavoo/pyntcloud/pull/321/files#diff-021377fee185787af295603466e9f1f3fec039246017a515213eb7c2f2115888

**Review by daavoo — APPROVED** [review]

**Pull request merged**